itests/kit/ensemble.go

```diff
@@ -603,12 +601,11 @@ func (n *Ensemble) Start() *Ensemble {
 			n.t.Fatalf("invalid config from repo, got: %T", c)
 		}
 		cfg.Common.API.RemoteListenAddress = m.RemoteListener.Addr().String()
-		cfg.Subsystems.EnableMarkets = m.options.subsystems.Has(SMarkets)
 		cfg.Subsystems.EnableMining = m.options.subsystems.Has(SMining)
 		cfg.Subsystems.EnableSealing = m.options.subsystems.Has(SSealing)
 		cfg.Subsystems.EnableSectorStorage = m.options.subsystems.Has(SSectorStorage)
 		cfg.Subsystems.EnableSectorIndexDB = m.options.subsystems.Has(SHarmony)
-		cfg.Dealmaking.MaxStagingDealsBytes = m.options.maxStagingDealsBytes
+		cfg.EnableLibp2p = true
```

Comment:
Is this right? it's `false` elsewhere by default now, can we remove this or switch it off and `false` the second argument to `node.StorageMiner()` below like it is for normal node instantiation?

I'm unsure what the libp2p functionality is for in the miner with markets gone now, maybe it can go all together, including that second argument to `node.StorageMiner()`?

Reply:
You are right. We don't need libp2p in the Miner anymore. This is na vestige of deal making that we no longer need.